hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobService.java

```diff
@@ -1317,6 +1312,55 @@ public Response retryLogAggregation(@PathParam("appId") String appId,
         json).build();
   }
 
+
+  /**
+   * Remove scheduling for the job with this jobid. The return value is a
+   * JSON object stating operation successful
+   * or not.
+   * <p>
+   * @param jobId
+   * @param sc
+   * @param req
+   * @return
+   */
+  @DELETE
+  @Path("/{jobId}/unschedule")
```

Comment:
@robzor92 path should be `/{jobId}/schedule`. Delete action is applied on a resource, the uri should not end in an action. Actions should be in query params or in the payload.

Similarly in the v2 the updateSchedule path will change from 
```
@POST
@Path("/updateschedule/{jobId}")
```
to
```
@POST
@Path("/{jobId}/schedule")
```

Reply:
Ok sure. Had a look in 510 and it is named jobId/*scheduled*. I will change the path and add queryparam.

Reply:
It will be refactoring as part of HOPSWORKS-510